cmd/host.go

```diff
@@ -44,6 +44,9 @@ CORE CONTROLS
 	- RETHINK_TLS_CERT: A pem encoded TLS certificate passed as string. Can be used instead of RETHINK_TLS_CERT_PATH.
 		Example: RETHINK_TLS_CERT_PATH="-----BEGIN CERTIFICATE-----\nMIIDZTCCAk2gAwIBAgIEV5xOtDANBgkqhkiG9w0BAQ0FADA0MTIwMAYDVQQDDClP..."
 
+  - Redis: If DATABASE_URL is a DNS starting with redis:// Redis will be used as a storage backend.
+		Example: DATABASE_URL=redis://x:password@host:123/0
```

Comment:
I think it would be better if the port are actually the defaults of the databases (same goes for mysql/postgres/rethink). I will fix it for those three, but could you replace `:123` with the redis default port?